.kokoro/publish_javadoc11.cfg

```diff
@@ -0,0 +1,21 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+build_file: "spring-cloud-gcp/.kokoro/publish_javadoc11.sh"
+
+# cloud-rad dev
+env_vars: {
+  key: "STAGING_BUCKET_V2"
+  value: "docs-staging-v2-dev"
+}
+
+before_action {
+  fetch_keystore {
+    keystore_resource {
+      keystore_config_id: 73713
+      keyname: "docuploader_service_account"
+    }
+  }
+}
```

Comment:
I think this config should live in g3.

Reply:
copied from discussion: `before_action` is only configurable in `build.proto` not `job.proto`.